### DISCUSSION TITLE
[@types/vimeo]: npm package vimeo has moved to @vimeo/vimeo

### DISCUSSION THREAD

**Discussion by leon** [DefinitelyTyped/DefinitelyTyped#67543] [Issues with a @types package]
I'm getting `vimeo@2.3.1: This package was migrated. For the latest features, run: npm i @vimeo/vimeo.`

Should we deprecate the @types/vimeo and publish a new @types/vimeo__vimeo?

[a comment by an automated account is not shown]

**Comment by mattleff** [comment]
@leon Sounds good to me! I don't work with any projects using `vimeo` anymore, but I'm happy to review if you send a PR. 👍🏻
